waku/v2/peermanager/peer_manager.go

```diff
@@ -351,9 +351,11 @@ func (pm *PeerManager) AddDiscoveredPeer(p service.PeerData, connectNow bool) {
 	if err == nil {
 		enr, err := pm.host.Peerstore().(wps.WakuPeerstore).ENR(p.AddrInfo.ID)
 		// Verifying if the enr record is more recent (DiscV5 and peer exchange can return peers already seen)
-		if err == nil && enr.Record().Seq() > p.ENR.Seq() {
+		if err == nil && enr.Record().Seq() >= p.ENR.Seq() {
 			return
 		}
+		pm.logger.Info("peer already found in peerstore, but re-adding it as ENR sequence is higher than locally stored",
```

Comment:
I wonder if this should be a .Debug?

Reply:
Thought about it and left it as Info as this should not be a frequent occurence for peers. 
Do you see a scenario where this log becomes bothersome?

Reply:
Not sure. I know that go-waku does a enr seq update as soon as it obtains an external address. Let's keep it on Info and see how it behaves :)